src/infrastructure/congestion.rs

```diff
@@ -0,0 +1,36 @@
+use crate:: {
+    Config,
+    net::{NetworkQuality, RttMeasurer},
+    sequence_buffer::{CongestionData, SequenceBuffer},
+};
+
+use std::time::Instant;
+
+/// Type responsible for keeping track of congestion information.
+pub struct CongestionHandler {
+    rtt_measurer: RttMeasurer,
+    rtt: f32,
+    congestion_data: SequenceBuffer<CongestionData>,
+    _quality: NetworkQuality,
```

Comment:
Why the _? Are we not using it anywhere?

Reply:
We already created this issue: #96